python/py-resumable-urlretrieve/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario resumable-urlretrieve 0.1.6
+name                py-${github.project}
+
+github.tarball_from tags
+
+platforms           darwin
+license             MIT
+
+python.versions     35 36 37
+
+maintainers         {gmail.com:rubendibattista @rubendibattista} openmaintainer
+description         Small library to fetch files over HTTP and resuming their download
+long_description    $description
+checksums           rmd160  afdd69d58743c1a506d8072d1a755abcbfaca038 \
+                    sha256  325a46500e2c7786d18ae254dc10b144b5ff2993e5afbf527cdfade50937a5ad \
+                    size    318654
+
+
+if {${subport} ne ${name}} {
+    depends_lib     port:py${python.version}-requests
```

Comment:
this needs ```py-setuptools``` as build dependency since it does ```from setuptools import setup``` in the ```setup.py```

Reply:
you added ```py-setuptools``` under ```depends_lib```, that's not correct. It is not needed at runtime, only during install; so as I said it should be added as ```depends_build-append```